src/pudl/extract/excel.py

```diff
@@ -1,12 +1,12 @@
 """Load excel metadata CSV files form a python data package."""
 import importlib.resources
 import pathlib
+import re
 from collections import defaultdict
 from io import BytesIO
 
 import dbfread
 import pandas as pd
-import regex as re
```

Comment:
This seems to have been an accident. The built-in `re` module does exactly the same thing.